docs/application-development/observability.md

```diff
@@ -58,67 +58,29 @@ For more information, see the [Go sample for metrics](https://github.com/tempora
 
-- Set the scope
-- Set the scrape end point
-- Initialize the Workflow service stub
-- Add metrics scope in the Workflow service stubs options
-- Initialize the Client
-
-In addition to the `client workflow workflowservicestubs workflowservicestubsoptions` options, set the following:
-
-```java
-import com.sun.net.httpserver.HttpServer;
-import com.uber.m3.tally.RootScopeBuilder;
-import com.uber.m3.tally.Scope;
-import com.uber.m3.util.ImmutableMap;
-import io.micrometer.prometheus.PrometheusConfig;
-import io.micrometer.prometheus.PrometheusMeterRegistry;
-import io.temporal.common.reporter.MicrometerClientStatsReporter;
-```
-
-The following code example demonstrates how to emit metrics from your Workflow.
-
-```java
-// task queue to be used for this sample
-  public static final String DEFAULT_TASK_QUEUE_NAME = "metricsqueue";
-
-  public static void main(String[] args) {
-
-    PrometheusMeterRegistry registry = new PrometheusMeterRegistry(PrometheusConfig.DEFAULT);
-```
-
-Set your scope.
-In this example, the scope will report once a second.
-
-```java
-Scope scope =
-        new RootScopeBuilder()
-            .reporter(new MicrometerClientStatsReporter(registry))
-            .reportEvery(com.uber.m3.util.Duration.ofSeconds(1));
-```
-
-Next, set an endpoint to allow Prometheus to scrape metrics from. In this example, the endpoint is set to a http server at `8080`.
-
-```java
-HttpServer scrapeEndpoint = MetricsUtils.startPrometheusScrapeEndpoint(registry, 8080);
-```
-
-Stopping the Worker will stop the http server that exposes the scrape endpoint.
-
-```java
-    Runtime.getRuntime().addShutdownHook(new Thread(() -> scrapeEndpoint.stop(1)));
-```
-
-Add the metrics scope to Workflow service stub options.
+The following example shows how to set the `MicrometerClientStatsReporter` for Prometheus and define the metrics scope with the `WorkflowServiceStubsOptions`.
 
 ```java
- WorkflowServiceStubsOptions stubOptions =
-        WorkflowServiceStubsOptions.newBuilder().setMetricsScope(scope).build();
+//...
+   // Set up Prometheus registry and stats reported
+   PrometheusMeterRegistry registry = new PrometheusMeterRegistry(PrometheusConfig.DEFAULT);
+   StatsReporter reporter = new MicrometerClientStatsReporter(registry);
+    // Set up a new scope, report every 10 seconds
+     Scope scope = new RootScopeBuilder()
+             .reporter(reporter)
+             .reportEvery(com.uber.m3.util.Duration.ofSeconds(10));
+   // Start the Prometheus scrape endpoint for metrics
+   HttpServer scrapeEndpoint = MetricsUtils.startPrometheusScrapeEndpoint(registry, 8081);
```

Comment:
We can't reference MetricsUtils here because this class is not a part of JavaSDK. This class is in samples.

Instead, I would remove this line altogether at least for now, and give a link to the pages that I sent you earlier about scraping endpoints and setting up Micrometer + Prometheus. 
Strictly speaking, it's not Temporal business to provide a lot of advice about how to setup Micrometer.

These is the link that I sent earlier:
https://tanzu.vmware.com/developer/learningpaths/application-observability/exposing-metrics-java-prometheus/